app/cdap/components/AddTransformation/SelectMultipleColumnList/index.tsx

```diff
@@ -0,0 +1,273 @@
+import {
+  Box,
+  Checkbox,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  FormControlLabel,
+  Typography,
+} from '@material-ui/core';
+import React, { useEffect, useRef, useState } from 'react';
+import { useStyles } from '../styles';
+import { SearchIcon } from '../iconStore';
+import { prepareDataQualtiy } from '../CircularProgressBar/utils';
+import DataQualityProgress from '../CircularProgressBar';
+import { NoDataSVG } from 'components/GridTable/iconStore';
+import { multipleColumnSelected } from '../constants';
+import T from 'i18n-react';
+
+const SelectColumnsList = (props) => {
+  const {
+    directiveFunctionSupportedDataType,
+    selectedColumnsCount,
+    columnData,
+    setSelectedColumns,
+    dataQuality,
+    functionName,
+  } = props;
+  const [columns, setColumns] = useState(columnData);
+  const [dataQualityValue, setDataQualityValue] = useState(dataQuality);
+  const [selectedColumns, setSelectedColumn] = useState([]);
+  const [focused, setFocused] = useState(false);
+  const classes = useStyles();
+  const ref = useRef(null);
+  const no_match =
```

Comment:
add type to no_match, use camelCase for variable names and add proper meaningful name

Reply:
Done